code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -914,17 +914,33 @@
 	..()
 	. = 1
 
-/datum/reagent/medicine/stimulants
-	name = "Stimulants"
-	id = "stimulants"
-	description = "Increases stun resistance and movement speed in addition to restoring minor damage and weakness. Overdose causes weakness and toxin damage."
+/datum/reagent/medicine/fervoxazine
+	name = "Fervoxazine"
+	id = "fervoxazine"
+	description = "Increases stun resistance and movement speed in addition to restoring minor damage and weakness. Overdose causes weakness and toxin damage. This reagent is even more potent with Golems"
 	color = "#78008C"
 	metabolization_rate = 0.5 * REAGENTS_METABOLISM
 	overdose_threshold = 60
 
-/datum/reagent/medicine/stimulants/on_mob_add(mob/living/L)
+/datum/reagent/medicine/fervoxazine/on_mob_add(mob/living/L)
+	..()
+	if(isgolem(L)) //More potent for golems
+		L.add_movespeed_modifier(id, update=TRUE, priority=100, multiplicative_slowdown = -2.5, blacklisted_movetypes=(FLYING|FLOATING))
+	else
+		L.add_movespeed_modifier(id, update=TRUE, priority=100, multiplicative_slowdown = -0.75, blacklisted_movetypes=(FLYING|FLOATING))
+
+/datum/reagent/drug/fervoxazine/on_mob_life(mob/living/carbon/M)   
+	if(prob(5))
+		var/high_message = pick("You feel heated and ready to move.", "You feel like a burning star.", "You feel like a meteor.")
+		to_chat(M, "<span class='notice'>[high_message]</span>")
+	M.adjust_bodytemperature(BODYTEMP_HEATING_MAX) //hot hot hot
+	M.adjustStaminaLoss(-2, 0) 
```

Comment:
-2 stamina/tick :joy: :joy: :joy:

Reply:
its a meth gun for golems i dont know what you expected

Reply:
all the antistun chems have outdated staminaheal as -2 is even less than natural regen

Reply:
ill fix